pages/doc/wavefront_prometheus.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Using Prometheus Queries in Wavefront
+keywords: query language
+sidebar: doc_sidebar
+published: true
+permalink: wavefront_prometheus.html
+summary: Run Prometheus from the Wavefront query line
+---
+
+Starting with release 2020.26, Wavefront includes support for Prometheus queries.
+* Run Prometheus queries directly from the Wavefront query line. The corresponding chart shows the information you'd expect.
+* Wavefront query language includes new functions and operators that allow you to "translate" any Prometheus query to a ts() query and use it with the rich set of existing ts() query language functions.
+
+## How to Run Prometheus Queries
+
+1. Create or edit a chart.
+2. Start typing your Prometheus query .
+3. Make changes to the visualization.
+  * See [Create and Customize Charts](ui_charts.html) for an intro.
+  * See [Chart Reference](ui_chart_reference) for details.
+
+![Prometheus query](images/prometheus_sample.png)
+
+## How Prometheus Queries Can Differ in Wavefront.
+
+Wavefront supports all Prometheus functions out of the box. There are a small number of differences.
```

Comment:
This isn't true.  We support most functions and operators but not all.  Check with Hao about the exact number.

Reply:
Changed to "most".